.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,28 @@
+name: golangci-lint
+on:
+  pull_request:
+    branches:
+      - master
+
+permissions:
+  contents: read
+
+concurrency:
+  group: '$${{ github.workflow }}-$${{ github.head_ref || github.ref }}'
+  cancel-in-progress: true
+
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
```

Comment:
GitHub-owned actions like these two can be pinned by version instead of hash - what would you think about switching to that?

Reply:
Sure - Some repos prefer using HASH, but I'm happy to switch to versions. :)